docs/tutorials/forum-application/1.abci-intro.md

```diff
@@ -0,0 +1,41 @@
+# Introduction to ABCI 2.0
+
+`ABCI 2.0` is an updated version of the ABCI (Application Blockchain Interface) from CometBFT. For more details, check the [specification document](https://docs.cometbft.com/v1.0/spec/abci/).
+
+![ABCI 2.0](images/ABCI2.jpg)
+
+## ABCI 2.0 Methods
+
+### PrepareProposal
+
+- **Purpose**: Modify block proposals before finalization.
+- **Process**: The block proposer collects pending transactions, creating a "raw proposal." The application can then reorder, add, or remove transactions, creating a "prepared proposal."
+
+### ProcessProposal
+
+- **Purpose**: Validate a block proposal based on application-specific rules.
+- **Process**: Validators assess the proposal and can reject invalid blocks. If rejected, the consensus algorithm will perform a nil prevote. The logic must be deterministic to ensure consistent outcomes.
```

Comment:
```suggestion
- **Process**: Validators assess the proposal and can reject invalid blocks. If rejected, the consensus algorithm will perform a nil prevote. The logic must be deterministic to avoid liveness problems.
```